src/v2/Apps/ViewingRoom/ViewingRoomApp.tsx

```diff
@@ -67,6 +73,21 @@ const ViewingRoomApp: React.FC<ViewingRoomAppProps> = ({
   return (
     <>
       <ViewingRoomMeta viewingRoom={viewingRoom} />
+      {showPreview && (
+        <Box position="fixed" left={0} top={58} width="100%" zIndex={1}>
+          <Banner backgroundColor="black10" textColor="black100">
+            <ClosedEyeIcon />
+            <Text
+              display="inline"
+              verticalAlign="top"
+              px={0.5}
+              lineHeight="18px"
+            >
+              This is a preview of your viewing room.
```

Comment:
**thought (non-blocking):** what if instead of a banner we made this like a little hovering square in one corner and said something like "This is a preview of your viewing room. It is scheduled to open Oct. 10, 2020" (if it's scheduled)? Could solve both the multiple-banner issue and could be nice QOL for partners to be reminded of when it opens.

*format via [Conventional Comments](https://conventionalcomments.org/) + [saved replies](https://github.com/settings/replies)*

Reply:
I think it's a legit idea. I'll let @tinn to be the final call here. The banner was a design from him.